__tests__/pages/aboutpage.test.tsx

```diff
@@ -1,11 +1,16 @@
-import About from "../../app/about/page";
 import { render } from "@testing-library/react";
+import About from "app/about/page";
 import { axe } from "jest-axe";
 import React from "react";
 
+jest.mock("next/navigation", () => ({
+  useRouter: jest.fn(),
+  usePathname: jest.fn(),
+}));
+
 describe("About page accessibility", () => {
   it("passes axe accessibility test", async () => {
     const { container } = render(<About />);
     expect(await axe(container)).toHaveNoViolations();
-  }, 15000);
+  }, 60000);
```

Comment:
Oiy, that's a big timeout increase.

Reply:
They don't take this long locally but I just want the test hook to not fail bc of timeout